app/components/authorShowHeader/authorShowHeader.scss

```diff
@@ -134,16 +135,14 @@
 .contactIconWrapper {
   display: inline-flex;
   width: 24px;
-  height: 24px;
+  vertical-align: middle;
```

Comment:
This is flex-layout element.
If you want to make centered vertical-align one, try to use `align-items: center`

Reply:
https://css-tricks.com/snippets/css/a-guide-to-flexbox/
this is the ultimate guide of Flexbox layout API.